### ISSUE TITLE
needs tests!

### ISSUE THREAD

**Issue by jklymak** [jklymak/pyglider#11]
We probably should run CI with the code on a few example data sets.

**Comment by jklymak** [comment]
Mostly added in #13 which is a smoketest on the two glider types.  If we add more glider examples, i.e. DFO, then we can run those as well.  

Todo:

- [ ] could make webpage and check that it builds properly?  Not sure if we have a way to serve webpage here, versus on CircleCI.
- [ ] could have a check that the files are present in the subdirs

**Comment by richardsc** [comment]
I'm not familiar with python packages, but a common approach in the R world is to make a webpage from the package that gets served by Github pages (e.g. https://dankelley.github.io/oce/). It is also possible to do the webpage build as a Github Action, meaning that it's automatic after pushes or PR acceptance.

**Comment by jklymak** [comment]
Yeah, thats different than the webpage the processing makes.  Our processing makes a webpage out of the figs directory and the yml and copies that somewhere for serving on https://cproof.uvic.ca  

What you are talking about we could add as well, which is a `doc` directory that makes a jekyll page or readthedocs page and is served by GitHub.  But... this is a pretty lightweight package, so I think the Readme is fine for now....

**Comment by richardsc** [comment]
Ah, that makes sense. I'll have to look into the webpage building -- that could be handy for us to as we transition to using pyglider for realtime and delayed data processing.
